torch/csrc/api/include/torch/nn/cloneable.h

```diff
@@ -59,7 +59,7 @@ class Cloneable : public virtual Module {
         "Are you sure you called register_buffer() inside reset() "
         "and not the constructor?");
     for (const auto& buffer : buffers_) {
-      auto data = autograd::Variable(*buffer).data().clone();
+      auto data = autograd::Variable(*buffer).clone();
```

Comment:
It's okay to remove `.data()` here, because `data` is being used in the `set_data(...)` call below, and `set_data(...)` ignores `data`'s AutogradMeta.